src/app/test/test-component-support.class.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, OnChanges } from '@angular/core';
+import { TestComponentSupport } from './test-component-support.class';
+import { TestBed, inject } from '@angular/core/testing';
+
+let testComponentSupport;
+let testComponentSupportSimple;
+
+@Component({
+  template: `<div
+                app-attr="abc"
```

Comment:
we should probably put this in a `html` file, or at least format the template to be consistent with the rest of the html spacing/formatting. Also, let's not use `<div>` - it's an anti-pattern and I don't want to encourage it's use unless there is a very, very good reason. So we can use `<p>` or `<section>` or something like that instead.